functions/bfc-functions.php

```diff
@@ -124,6 +124,121 @@ function bfc_nouveau_has_nav( $args = array() ) {
 	return true;
 }
 
+/* Main Nav Menu */
+function bfc_top_nav_menu_builder() {
+$menuitemarray = array (
```

Comment:
Fixed. I found several lines that I introduced that had a mixture of spaces and tabs. I went through every line I added and forced the correct number of tabs to start each line. If you still see indentation problems it might mean that my editor config still isn't quite right.